libwayshot/src/lib.rs

```diff
@@ -494,19 +548,32 @@ impl WayshotConnection {
         })
     }
 
+    /// Take a screenshot from the specified region.
+    pub fn screenshot(
+        &self,
+        capture_region: CaptureRegion,
+        cursor_overlay: bool,
+    ) -> Result<RgbaImage> {
+        self.screenshot_region_capturer(RegionCapturer::Region(capture_region), cursor_overlay)
```

Comment:
Typo?

Reply:
I don't think so? `RegionCapturer` is a new enum to differentiate what we're capturing:

```
pub enum RegionCapturer {
    Outputs(Vec<OutputInfo>),
    Region(CaptureRegion),
    Freeze(Box<dyn Fn() -> Result<CaptureRegion>>),
}
```

Still need to do some documenting however to make this and the reasoning for it more clear!

Reply:
Can you add a example above it ?

Reply:
Will try to add an example of the freeze functionality. :)